src/styles/card.css

```diff
@@ -0,0 +1,99 @@
+@import "./reset.css";
+
+@import "./stars.css";
+
+/* #region Card */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  background: #e5e5e5;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 400px;
```

Comment:
[STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.